File: Std/JSScript/ALittle.js
{


__ALITTLEAPI_Std_Init = async function(base_path, callback) {
	await Require(base_path + "Std/Reflect/ReflectCmd");
	await Require(base_path + "Std/Reflect/ReflectCsv");
	await Require(base_path + "Std/Reflect/ReflectMessage");
	await Require(base_path + "Std/Utility/IHeapTimer");
	{
		await Require(base_path + "Adapter/JavaScript/Timer");
	}
	await Require(base_path + "Std/Loop/LoopObject");
	await Require(base_path + "Std/Loop/ILoopSystem");
	await Require(base_path + "Std/Loop/LoopSystem");
	await Require(base_path + "Std/Loop/LoopFunction");
	await Require(base_path + "Std/Loop/LoopGroup");
	await Require(base_path + "Std/Loop/LoopList");
	await Require(base_path + "Std/Loop/LoopTimer");
	await Require(base_path + "Std/Utility/CsvConfig");
	await Require(base_path + "Std/Utility/EventDispatcher");
	await Require(base_path + "Std/Utility/SafeIDCreator");
	await Require(base_path + "Std/Utility/String");
	await Require(base_path + "Std/Utility/File");
	await Require(base_path + "Std/Utility/ConfigSystem");
	await Require(base_path + "Std/Net/HttpFileReceiverTemplate");
	await Require(base_path + "Std/Net/HttpFileSenderTemplate");
	await Require(base_path + "Std/Net/HttpReceiverTemplate");
	await Require(base_path + "Std/Net/HttpSenderTemplate");
	await Require(base_path + "Std/Net/MsgCommonTemplate");
	await Require(base_path + "Std/Net/MsgReceiverTemplate");
	await Require(base_path + "Std/Net/MsgSenderTemplate");
	await Require(base_path + "Std/Net/MsgSessionTemplate");
	if (callback !== undefined) {
		callback();
	}
}

}

File: ALittle/Module/Std/Script/ALittle.lua
-- ALittle Generate Lua
local ___pairs = pairs
local ___ipairs = ipairs


function RequireStd(base_path)
local ___COROUTINE = coroutine.running()
	ALittle.Require(base_path, "Std/Reflect/ReflectCmd")
	ALittle.Require(base_path, "Std/Reflect/ReflectCsv")
	ALittle.Require(base_path, "Std/Reflect/ReflectMessage")
	ALittle.Require(base_path, "Std/Loop/LoopObject")
	ALittle.Require(base_path, "Std/Loop/ILoopSystem")
	ALittle.Require(base_path, "Std/Loop/LoopFrame")
	ALittle.Require(base_path, "Std/Loop/LoopFunction")
	ALittle.Require(base_path, "Std/Loop/LoopGroup")
	ALittle.Require(base_path, "Std/Loop/LoopList")
	ALittle.Require(base_path, "Std/Loop/LoopTimer")
	ALittle.Require(base_path, "Std/Utility/Bit")
	ALittle.Require(base_path, "Std/Utility/WeakRef")
	ALittle.Require(base_path, "Std/Utility/IHeapTimer")
	ALittle.Require(base_path, "Std/Utility/ISchedule")
	ALittle.Require(base_path, "Std/Utility/CsvConfig")
	ALittle.Require(base_path, "Std/Utility/EventDispatcher")
	ALittle.Require(base_path, "Std/Utility/SafeIDCreator")
	ALittle.Require(base_path, "Std/Utility/String")
	ALittle.Require(base_path, "Std/Utility/File")
	ALittle.Require(base_path, "Std/Utility/JsonConfig")
	ALittle.Require(base_path, "Std/Net/HttpFileReceiverTemplate")
	ALittle.Require(base_path, "Std/Net/HttpFileSenderTemplate")
	ALittle.Require(base_path, "Std/Net/HttpReceiverTemplate")
	ALittle.Require(base_path, "Std/Net/HttpSenderTemplate")
	ALittle.Require(base_path, "Std/Net/MsgCommonTemplate")
	ALittle.Require(base_path, "Std/Net/MsgReceiverTemplate")
	ALittle.Require(base_path, "Std/Net/MsgSenderTemplate")
	ALittle.Require(base_path, "Std/Net/MsgSessionTemplate")
	ALittle.Require(base_path, "Adapter/Lua/LuaHeapTimer")
	ALittle.Require(base_path, "Adapter/Lua/LuaCsvFile")
	if net ~= nil then
		ALittle.Require(base_path, "Adapter/Lua/LuaHttpFileInterface")
		ALittle.Require(base_path, "Adapter/Lua/LuaHttpInterface")
		ALittle.Require(base_path, "Adapter/Lua/LuaMsgInterface")
		ALittle.Require(base_path, "Adapter/Lua/LuaMessageFactory")
		ALittle.Require(base_path, "Adapter/Lua/LuaSchedule")
	end
	ALittle.Require(base_path, "Std/Loop/LoopSystem")
	ALittle.Require(base_path, "Std/Utility/Schedule")
end

